File: 백준/Bronze/1919. 애너그램 만들기/애너그램 만들기.py
def cnt_alpha(string):
    result = [0 for _ in range(26)]
    base = ord('a')
    for c in string:
        result[ord(c) - base] += 1
    return result

a = input()
b = input()

cnt_a = cnt_alpha(a)
cnt_b = cnt_alpha(b)

ans = 0
for i in range(26):
    ans += abs(cnt_a[i] - cnt_b[i])
    
print(ans)

File: 백준/Bronze/1919. 애너그램 만들기/애너그램 만들기.cc
#include <iostream>
#include <string>
#include <algorithm>

using namespace std;

void cnt_alpha(const string &str, int cnt[26]) {
    for (char c: str) {
        cnt[c - 'a']++;
    }
}

int main() {
    string a, b;
    
    cin >> a >> b;
    
    int cnt_a[26] = {};
    int cnt_b[26] = {};
    
    cnt_alpha(a, cnt_a);
    cnt_alpha(b, cnt_b);
    
    int ans = 0;
    for (int i = 0; i < 26; i++) {
        ans += abs(cnt_a[i] - cnt_b[i]);
    }
    cout << ans;
    return 0;
}
